test/preview/pipeline/test_pipeline.py

```diff
@@ -28,29 +29,236 @@ def test_pipeline_store_api():
 
 
 @pytest.mark.unit
-def test_pipeline_stores_in_params():
+def test_pipeline_storemixin_one_existing_docstore():
     store_1 = MockStore()
     store_2 = MockStore()
 
     @component
-    class MockComponent:
+    class MockComponent(StoreMixin):
         @dataclass
         class Input(ComponentInput):
             value: int
-            stores: Dict[str, Any]
 
         @dataclass
         class Output(ComponentOutput):
             value: int
 
         def run(self, data: Input) -> Output:
-            assert data.stores == {"first_store": store_1, "second_store": store_2}
+            assert self.store == store_1
```

Comment:
I wouldn't `assert` this here, let's keep it in the body of the test.

Reply:
Same thing for all the other `assert`s in other tests.

Reply:
It makes it less maintainable in my opinion, and also a bit hidden. I'd expect that in the future we'll have a sort of factory to build these mocks and it would be quite easy to lose that `assert` when making that change.

Reply:
Done :heavy_check_mark: